Dockerfile

```diff
@@ -1,6 +1,10 @@
-FROM progrium/busybox:latest
+FROM gliderlabs/alpine:3.6
 
-MAINTAINER Bjoern Gruening <bjoern.gruening@gmail.com>
+LABEL author="Roberto Spreafico <roberto.spreafico@gmail.com>"
+LABEL maintainer="Bjoern Gruening <bjoern.gruening@gmail.com>"
+
+RUN apk add --no-cache procps bash
+
+SHELL ["/bin/bash", "-c"]
```

Comment:
I'm a little hesitant to change the default for so many images. Do you really need this?

Reply:
We do need `procps` for Nextflow compatibility - this was the original reason triggering the overhaul of Biocontainers' base image. `bash` is added as it was included in the previous image too. As for the default shell, I don't have very strong opinions there. I cannot think at anything running on `sh` that wouldn't run on `bash` as well, but I understand the hesitation.

Reply:
I was only talking about the bin/bash here :)